composer.json

```diff
@@ -60,8 +60,8 @@
         "phpunit": "phpunit --verbose --colors=always",
         "phpcs": "phpcs src tests --standard=psr2 -sp --colors",
         "phpstan": [
-            "phpstan analyse -c phpstan.neon src --level 2 --no-progress",
-            "phpstan analyse -c phpstan-tests.neon tests --level 2 --no-progress"
+            "php vendor/phpstan/phpstan/bin/phpstan analyse -c phpstan.neon src --no-progress",
+            "php vendor/phpstan/phpstan/bin/phpstan analyse -c phpstan-tests.neon tests --no-progress"
```

Comment:
Why the change from `phpstan` to the whole path? It should work the old way.

Reply:
`phpstan` uses whichever version of phpstan is found under `$PATH`
`vendor/bin/phpstan` would work, except for some windows environments, i.e. git bash
`vendor/phpstan/phpstan/bin/phpstan` refers specifically to the version required by the composer.json

Reply:
It should always use the one installed through the Composer, see [the docs](https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands).

> Note: Before executing scripts, Composer's bin-dir is temporarily pushed on top of the PATH environment variable so that binaries of dependencies are easily accessible. In this example no matter if the phpunit binary is actually in vendor/bin/phpunit or bin/phpunit it will be found and executed.

In what situation it did not work as expected?

Reply:
> It should always use the one installed through the Composer, see the docs.

might be an artefact of past experience/earlier development environments. checking now.

Reply:
yup, I stand corrected - always nice to learn new things :)

Reply:
Great! I was afraid it is actually broken (I use similar configuration elsewhere)